### PR TITLE
Added lazy creation of framebufferobject

### PR DESCRIPTION
**Pull request by christopherlagerhult**

### CODE REVIEW

File: source/draw/gpu/opengl/OpenGLPacked.ooc [hunk 1]
```diff
@@ -23,21 +23,29 @@ OpenGLPacked: abstract class extends GpuImage {
 	channels ::= this _channels
 	context ::= this _context as OpenGLContext
 	recyclable ::= this _recyclable
-	_renderTarget: GLFramebufferObject
+	_frameBufferObjectHandle: GLFramebufferObject
```
Comment (emilwestergren) [inline source/draw/gpu/opengl/OpenGLPacked.ooc]: ` = null` ?

File: source/draw/gpu/opengl/OpenGLPacked.ooc [hunk 2]
```diff
@@ -23,21 +23,29 @@ OpenGLPacked: abstract class extends GpuImage {
 	channels ::= this _channels
 	context ::= this _context as OpenGLContext
 	recyclable ::= this _recyclable
-	_renderTarget: GLFramebufferObject
+	_frameBufferObjectHandle: GLFramebufferObject = null
+	_renderTarget: GLFramebufferObject {
+		get {
+			if (!_frameBufferObjectHandle)
```
Comment (marcusnaslund) [inline source/draw/gpu/opengl/OpenGLPacked.ooc]: missing `this`

File: source/draw/gpu/opengl/OpenGLPacked.ooc [hunk 3]
```diff
@@ -23,21 +23,29 @@ OpenGLPacked: abstract class extends GpuImage {
 	channels ::= this _channels
 	context ::= this _context as OpenGLContext
 	recyclable ::= this _recyclable
-	_renderTarget: GLFramebufferObject
+	_frameBufferObjectHandle: GLFramebufferObject = null
+	_renderTarget: GLFramebufferObject {
+		get {
```
Comment (marcusnaslund) [inline source/draw/gpu/opengl/OpenGLPacked.ooc]: you can do this do save some space
```
_renderTarget: GLFramebufferObject { get {
   ...
}}
```

**Comment by marcusnaslund** [comment]
ready?

**Pull request merged**